tests/unit/test.program.js

```diff
@@ -323,4 +324,13 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
+  it('returns the development package version', () => {
+    process.env.NODE_ENV = 'development';
+    let root = path.join(__dirname, '..', '..');
```

Comment:
to reduce repetition, set this to the top of the `describe()` function for this group of tests so that it can be referenced by both tests